Tests/LayoutTests/UIKit/UIView+AutoLayoutTests.swift

```diff
@@ -259,6 +259,57 @@ final class UIViewAutoLayoutTests: XCTestCase {
                                                                 constant: 40)))
     }
 
+    func testSizeConstraintsWithWidthAndHeight() {
```

Comment:
I think this should match the method signature more closely to be in line with our naming conventions:

```suggestion
    func testSizeConstraintsWidthHeight() {
```

Reply:
I updated the other test method, but decided to leave this one as is since it more closely matches the other tests in this file.